packages/circuits/circuits/dataProof.circom

```diff
@@ -2,10 +2,11 @@ pragma circom 2.1.0;
 
 include "../../../node_modules/@unirep/circuits/circuits/hasher.circom";
 include "../../../node_modules/@unirep/circuits/circuits/incrementalMerkleTree.circom";
+include "../../../node_modules/@unirep/circuits/circuits/bigComparators.circom";
 include "../../../node_modules/@unirep/circuits/circuits/circomlib/circuits/comparators.circom";
 
 
-template DataProof(STATE_TREE_DEPTH, FIELD_COUNT, SUM_FIELD_COUNT) {
+template DataProof(STATE_TREE_DEPTH, FIELD_COUNT, SUM_FIELD_COUNT, REPL_NONCE_BITS) {
```

Comment:
Though it inherits unirep config
but in this template, we can still check 
```circom
assert(SUM_FIELD_COUNT < FIELD_COUNT);
```
to make sure `FIELD_COUNT - SUM_FIELD_COUNT` won't be overflowed